Build.mak

```diff
@@ -0,0 +1,16 @@
+# Set of flags required to compile vibe.d without dub
+override DFLAGS += -version=VibeLibeventDriver -version=Have_openssl \
+	-I./submodules/vibed/source -I./submodules/libevent -I./submodules/openssl
+
+# Compile all vibe.d as a single static library because rdmd can't track
+# all imported dependencies correctly in this case
+$O/libvibed.a: $(shell find $C/submodules/vibed/source -type f -name '*.d')
```

Comment:
Maybe then add that bit about compilation time to the comment? (and maybe even a link to the issue about dependencies) :innocent: 

> In general it can't ever be 100% relied on

Is quite hard to believe the compiler don't know which files are included by a program :)